File: testdata/jennies/rawtypes/refs/PHPRawTypes/src/Types/Refs/SomeStruct.php
<?php

namespace Grafana\Foundation\Types\Refs;

class SomeStruct implements \JsonSerializable {
    /**
     * @var mixed
     */
    public $fieldAny;

    /**
     * @param mixed|null $fieldAny
     */
    public function __construct( $fieldAny = null)
    {
        $this->fieldAny = $fieldAny ?: null;
    }

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "FieldAny" => $this->fieldAny,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/maps/CSharpRawTypes/src/Grafana/Foundation/Maps/SomeStruct.cs
namespace Grafana.Foundation.Maps;


public class SomeStruct
{
    public object FieldAny;

    public SomeStruct()
    {
        this.FieldAny = new object();
    }

    public SomeStruct(object fieldAny)
    {
        this.FieldAny = fieldAny;
    }
}
